spaces/golff/index.json

```
@@ -0,0 +1,15 @@
+{
+  "key": "gof",
```

The key of your space must also be the folder name and skin class name.
```suggestion
  "key": "golff",
```